c++/configure.ac

```diff
@@ -1,6 +1,6 @@
 ## Process this file with autoconf to produce configure.
 
-AC_INIT([Capn Proto],[0.9-dev],[capnproto@googlegroups.com],[capnproto-c++])
+AC_INIT([Capn Proto],[0.8.0-rc1],[capnproto@googlegroups.com],[capnproto-c++])
```

Comment:
Oops, it looks like you based this PR off the release branch, but are targeting the PR at the master branch. The version in the master branch is now 0.9-dev. Once this is merged into master we can potentially cherry-pick it into a release branch, but please don't change the version in master. (This is causing the test failures.)